### ISSUE TITLE
Show stress-strain curves

### ISSUE THREAD

**Issue by siemdejong** [siemdejong/skinstression#65]
During training, use the validation set for calculating curves in case of 3 variables.

If only training for k, try plotting a linear line going through the inflection point at xc.

Make a script that uses the prediction dataloader for this.
